Packs/PrismaCloud/Integrations/RedLock/RedLock.py

```diff
@@ -548,15 +555,21 @@ def fetch_incidents():
     demisto.info("Executing Prisma Cloud (RedLock) fetch_incidents with payload: {}".format(payload))
     response = req('POST', 'alert', payload, {'detailed': 'true'})
     incidents = []
+    new_fetches = []
     for alert in response:
+        alert_id = alert.get('id')
+        if alert_id in last_fetches:
+            continue
+        new_fetches.append(alert_id)
```

Comment:
Let's review this offline, I believe the solution might create an issue if you fetch the same payload twice in a row (if for example there are more than `limit` alerts with the same alert time - In that case, you'll delete the prev run `last_fetches` and refetch it).